### DISCUSSION TITLE
Why are we creating yet another configuration file?

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [remix-run/react-router#12549] [Q&A]
I am a remix user and I want to port my app to use react-router but after reading the react-router documentation I noticed you brought back `remix.config.js` config file with just another name `react-router.config.js` . I thought we are done with this nonsense. Why can't you just let us pass configuration in `vite.config.js` file. I thought react-router is just a vite plugin. I have 4 configuration files already. WHY???

**Comment by LeoDanielsson** [comment]
You can read about the decision here:

https://github.com/remix-run/react-router/blob/main/decisions/0013-react-router-config-ts.md
